docker/Dockerfile.base

```diff
@@ -26,23 +23,15 @@ RUN /bin/sh -c 'curl https://sdk.cloud.google.com | bash' && \
     /google-cloud-sdk/bin/gcloud -q components install beta kubectl
 ENV PATH $PATH:/google-cloud-sdk/bin
 
-RUN wget -nv -O spark-2.4.0-bin-hadoop2.7.tgz https://archive.apache.org/dist/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz && \
```

Comment:
Why was this removed?

Reply:
Because I install pyspark using pip instead. Downloading spark manually and not pip-installing it means that all our packages that depend on pyspark will fail `pip check` because pyspark isn't actually pip installed.